e2e/utils/flows/addDataSource.ts

```diff
@@ -58,7 +58,7 @@ export const addDataSource = (config?: Partial<AddDataSourceConfig>) => {
 
   if (basicAuth) {
     cy.contains('label', 'Basic auth').scrollIntoView().click();
-    cy.contains('.gf-form-group', 'Basic Auth Details')
+    cy.contains('section', 'Basic Auth Details')
```

Comment:
The tests passed without these changes, so I don't think this is used in our e2e tests. Could be used in some plugins though.